daffy_lib.gemspec

```diff
@@ -18,16 +18,27 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
+  spec.add_development_dependency 'attr_encrypted'
+  spec.add_development_dependency 'aws-sdk-s3'
```

Comment:
Can remove this redundant dependency. This is a requirement of `porky_lib`, which you are specifying below.